api/serializers.py

```diff
@@ -0,0 +1,79 @@
+from rest_framework import serializers
+from rest_framework.parsers import JSONParser
+
+from game.models import Operation
+from run.models import Run
+from run.models import ParameterValue
+from storage.models import File
+from storage.serializers import FileSerializer
+
+
+# class FilePathSerializer(serializers.ModelSerializer):
+#     file = FileSerializer(many=False)
+#
+#     # definition = FileDefinitionSerializer()
+#
+#     class Meta:
+#         model = FilePath
+#         fields = ('file',
+#                   # 'definition',
+#                   )
+
+class ParameterValueSetSerializerField(serializers.Field):
+    def to_representation(self, value):
+        result = {}
+        for parameter_value in value:
+            # todo serialize file definition
+            result[parameter_value.parameter.name] = parameter_value._value
+        return result
+
+    def to_internal_value(self, data):
+        internal_value = []
+        if not isinstance(data, dict):
+            raise serializers.ValidationError('parameters data must be a dictionary.')
+            for parameter in data:
+                internal_value.append(
+                    ParameterValue(parameter=Parameter.objects.get(name=parameter), _value=data[parameter]))
```

Comment:
I think consensus was to let this model return list of tuples and create ParameterValue outside to avoid coupling.